feedbackfront/opgfeedbackfront/feedbackfront_blueprint.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+from flask import jsonify
+from flask import current_app
+import logging
```

Comment:
I think only line 1 is importing anything useful.

Reply:
Yep these were old imports from before. Now removed.